packages/terra-framework-docs/src/terra-dev-site/doc/embedded-content-consumer/example/BasicConsumer.jsx

```diff
@@ -8,7 +8,7 @@ Consumer.init();
 const BasicConsumer = () => (
   <EmbeddedContentConsumer
     src="/terra-framework/#/raw/provider/cerner-terra-framework-docs/embedded-content-consumer/providers/basic-provider"
-    options={{ iframeAttrs: { title: 'Basic content example' } }}
+    title="Basic content example"
```

Comment:
Do you still need to add options={{}} here?

Reply:
No, it's not needed. That will be undefined but the code in `EmbeddedContentConsumer` will check and add an empty object.

Reply:
It's optional as well.